pkgs/by-name/in/installShellFiles/setup-hook.sh

```diff
@@ -45,7 +43,9 @@ installManPage() {
         else
             outRoot=${!outputMan:?}
         fi
-        install -Dm644 -T "$path" "${outRoot}/share/man/man$suffix/$basename" || return
+        local outPath="${outRoot}/share/man/man$suffix/$basename"
+        install -Dm644 -T "$path" "$outPath" || \
+            { nixErrorLog "${FUNCNAME[0]}:" "install $path to $outPath failed"; return 1; }
```

Comment:
Curious why not the following:
```suggestion
        if ! install -Dm644 -T "$path" "$outPath"; then
            nixErrorLog "${FUNCNAME[0]}: install $path to $outPath failed"
            return 1
        fi
            
```

Alternatively, let the `install` tool output its errors: https://git.savannah.gnu.org/cgit/coreutils.git/tree/src/install.c shows that it's got substantial error logging.

Reply:
Good.